verifier-guest.sh

```diff
@@ -26,10 +26,20 @@ GEO_STABLE_HASH="1c65c9b"
 geonode_setup_env()
 {
     export PYTHONPATH=$HOME/oq-platform2:$HOME/oq-platform-taxtweb:$HOME/oq-platform-ipt
-    export DJANGO_SETTINGS_MODULE='openquakeplatform.settings'
+    #export DJANGO_SETTINGS_MODULE='openquakeplatform.settings'
```

Comment:
Remove commented export or comment it with the reason to keep it.

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/ef76e94f3ff4e6d8f8f90fc000b84f57588bf9b4